package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "boilerplate-web-app",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
A milestone of sorts: the first time we increment the version of this project.

Since this project isn't yet designed to expose a specific interface, the semver policy of this project for now is that we will increment the semver on DB schema changes or changes in the AJAX REST API exposed at the /api path.